AddLog.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo
```

Comment:
You can remove this 3 lines before submitting.

Reply:
I have copied those three lines from all our other scripts.  here is an example:
https://github.com/jamorham/nightscout-vps/blob/54e02516fde1005ce89e7a36477bb888bbd064cb/NS_Install.sh#L1-L3

Are those lines redundant in all of our scripts?

Reply:
I think that since this is just a small utility function, it does not add value to printing that this function is running